upload/upload.go

```diff
@@ -54,41 +53,63 @@ func loadConfigFile(path string) (*Config, error) {
 
 	cfg, err := ini.Load(path)
 	if err != nil {
-		return config, nil
+		return config, err
 	}
 
 	if err := cfg.Section("").MapTo(config); err != nil {
 		return nil, err
 	}
 
+	if config.AccessKey == "" || config.SecretKey == "" || config.AccessToken == "" {
+		return nil, errors.New("failed to find credentials in configuration file")
+	}
+
+	if config.HostBase == "" {
+		config.HostBase = "inbox.test.ega.nbis.se"
```

Comment:
This can not be hardcoded.

Reply:
Should I create a configuration then including the `HostBase` and `region`?

Reply:
Standalone SDA, federated EGA as well as all test sites need to be able to use this tool.
So anything that corresponds to host names, user ID and so on can not be but in the code.